example/pubspec.yaml

```
@@ -1,56 +1,19 @@
 name: permission_handler_example
 description: Demonstrates how to use the permission_handler plugin.
 
+environment:
+  sdk: ">=2.1.0 <3.0.0"
```

Sorry @long1eu, I missed this the first round. Could you please update the version here to (this is causing the CI to fail at the moment):

```suggestion
  sdk: ">=2.0.0 <3.0.0"
```